padertorch/contrib/examples/speech_enhancement/simple_mask_estimator/evaluate.py

```diff
@@ -0,0 +1,153 @@
+"""
+Evaluation script for the last mask estimator trained in $STORAGE_ROOT.
+Saves results to $STORAGE_ROOT/speech_enhancement/simple_mask_estimator_{id}/evaluate_{eval_id}
+mpiexec -np 8 python -m padertorch.contrib.examples.speech_enhancement.simple_mask_estimator.evaluate
+
+"""
+import os
+from pathlib import Path
+
+import dlp_mpi
+import numpy as np
+import paderbox as pb
+import padertorch as pt
+import pb_bss
+import torch
+from einops import rearrange
+from padercontrib.database import JsonAudioDatabase
+from padercontrib.database.chime import Chime3
+from padercontrib.database.iterator import AudioReader
+from pb_bss.extraction import get_bf_vector
+
+from . import SimpleMaskEstimator
+
+
+def change_example_structure(example):
+    stft = pb.transform.stft
+    audio_data = example['audio_data']
+    net_input = dict()
+    net_input['observation'] = audio_data['observation']
+    net_input['observation_stft'] = stft(
+        audio_data['observation']).astype(np.complex64)
+    net_input['observation_abs'] = np.abs(
+        net_input['observation_stft']).astype(np.float32)
+    net_input['speech_source'] = audio_data['speech_source']
+    net_input['example_id'] = example['example_id']
+    return net_input
+
+
+def get_test_dataset(database: JsonAudioDatabase):
+    # AudioReader is a specialized function to read audio organized
+    # in a json as described in pb.database.database
+    audio_reader = AudioReader(audio_keys=[
+        'observation', 'speech_source'
+    ])
+    val_iterator = database.get_dataset_test()
+    return val_iterator.map(audio_reader) \
+        .map(change_example_structure)
+
+
+def evaluate():
+    model = SimpleMaskEstimator(513)
+    task_dir = STORAGE_ROOT / 'speech_enhancement'
+    dirs = list(task_dir.glob('simple_mask_estimator_*'))
+    latest_id = sorted([int(path.name.split('_')[-1]) for path in dirs])[-1]
+    model_dir = task_dir / f'simple_mask_estimator_{latest_id}'
+    checkpoint_path = model_dir / 'checkpoints' / 'ckpt_best_loss.pth'
```

Comment:
Can you resolve the symlink and print something to stdout? e.g. `print(f'Start to evaluate the checkpoint {checkpoint_path} and will write the evaluation result to ...')`

Reply:
done